src/api/glue-api.js

```diff
@@ -0,0 +1,35 @@
+import axios from 'axios'
+// import qs from 'qs'
+
+const ROOT_URL = 'https://glue-api.zapimoveis.com.br/v2/listings?categoryPage=RESULT'
```

Comment:
I think that you maybe can move this const for a specific file of settings. What do you think?

Also, I think that `BASE_URL` is a better name.

Reply:
Something like
`src > settings > constants.js`
or
`src > settings > settings.js`
or
`src > setting.js`

About the name, I don't see much different between `ROOT` and `BASE`, for me base and root have the same meaning.

Reply:
I think this way is better:
`src > settings > constants.js`

- - - -

Your'r right. But is "common" we found `BASE_URL` for this context.

Some examples using base URL:
https://github.com/axios/axios#axioscreateconfig
https://github.com/axios/axios#request-config
https://github.com/olxbr/oz-assets-zap/blob/master/.lighthouserc.js#L1

Reply:
Goi it, gonna implement that.